astro/src/content/docs/lifecycle/authenticate-users/oauth/endpoints.mdx

```diff
@@ -724,79 +724,9 @@ _Response Codes_
 
 If you will be using the Refresh Token Grant, you will make a request to the Token endpoint to exchange the user's refresh token for an access token.
 
-<API method="POST" uri="/oauth2/token"/>
```

Comment:
Was this all duplicate?

Reply:
Yes. The entire `Refresh Token Grant Request` request portion was duplicated.